tests/integration/models/device.spec.js

```diff
@@ -1273,10 +1273,14 @@ describe('Device Model', function () {
 				});
 
 				describe('given device url is enabled', function () {
-					givenADevice(beforeEach);
+					givenADevice(before);
```

Comment:
did these changes sneak into this PR by accident?

Reply:
Just spotted them while fixing them and thought I should sneak then in :)
Moved to a different PR since I found a few more things to merge to make the tests faster :)
Thanks for the fast review @xginn8